packages/cli/fragments/fragment-angular/src/assets/tauri.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="206" height="231" viewBox="0 0 206 231" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
remove this file since it will be copied over from `_assets_`. Also remove `.gitkeep` file, it serves no purpose.